crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -172,11 +180,9 @@ where
             &schnorr_ver_key_preimage_vars,
             1,
         )?[0];
-        circuit.enforce_equal(
-            schnorr_ver_key_comm,
-            lightclient_state_var.stake_table_comm_var.1,
-        )?;
+        circuit.enforce_equal(schnorr_ver_key_comm, lightclient_state_var[5])?;
```

Comment:
Instead of using 5 (and 6 below), better to set it as a parameter, otherwise it's easy to lead to bugs if we change the `lightclient_state_var` later.

Reply:
it's not even gonna be parameter, there should be `struct StakeTableCommVar` and accessing to certain field should be clear from the getter name

Reply:
check https://github.com/EspressoSystems/HotShot/commit/599a75df2a83eeddfadb783d23cfccb30ad2d494